WebApiCoreSeed.Domain/Services/UserService.cs

```diff
@@ -30,75 +28,46 @@ public UserService(WebApiCoreSeedContext dbContext)
             return await _dbContext.Users.FindAsync(userId);
         }
 
-
         public async Task<List<User>> GetAsync()
         {
-            var Users = _dbContext.Users.ToList();
-            return Users;            
+            return await _dbContext.Users.ToListAsync();
         }
 
-        public async Task<string> CreateAsync(User user)
+        public async Task<int> CreateAsync(User user)
         {
-            string result = string.Empty;
+            user.CreatedOn = DateTime.Now;
             _dbContext.Users.Add(user);
-            _dbContext.SaveChanges();
-            result = CodeResponse.Sucessful.ToString();
-            return result;
-
+            return await _dbContext.SaveChangesAsync();

```

Comment:
Isn't `SaveChangesAsync` a `Task`? (void?)

Reply:
Nope, It returns the amount of affected rows. I'm not sure but I think it was void for EF 6